pizjuce/MiddyMorphy/ControllerGUI.cpp

```diff
@@ -126,132 +111,92 @@ void ControllerGUI::mouseUp (const MouseEvent& e)
         }
         else
         {
-            this->value->setValue (0, dontSendNotification);
+            this->value->setValue (0, juce::dontSendNotification);
         }
     }
     else
     {
-        this->value->setValue (controller->getInterpolatedValue(), dontSendNotification);
+        this->value->setValue (controller->getInterpolatedValue(), juce::dontSendNotification);
     }
-    this->channel->setValue (controller->getChannel(), dontSendNotification);
-    this->ccNo->setValue (controller->getCcNo(), dontSendNotification);
+    this->channel->setValue (controller->getChannel(), juce::dontSendNotification);
+    this->ccNo->setValue (controller->getCcNo(), juce::dontSendNotification);
     this->name->setText (controller->getName(), false);
     int state = controller->getState();
     switch (state)
     {
         case Controller::undefined:
-            value->setColour (Label::textColourId, Colours::blue);
+            value->setColour (juce::Label::textColourId, juce::Colours::blue);
             break;
         case Controller::mutival:
-            value->setColour (Label::textColourId, Colours::red);
+            value->setColour (juce::Label::textColourId, juce::Colours::red);
             break;
         case Controller::defined:
-            value->setColour (Label::textColourId, Colours::black);
+            value->setColour (juce::Label::textColourId, juce::Colours::black);
             break;
     }
 
     this->repaint();
-    // Bouml preserved body end 0003F38D
 }
 
-void ControllerGUI::textEditorTextChanged (TextEditor& editor)
+void ControllerGUI::textEditorTextChanged (juce::TextEditor& editor)
 {
-    // Bouml preserved body begin 0004178D
     if (&editor == name)
     {
         controller->setName (editor.getText());
     }
-    // Bouml preserved body end 0004178D
 }
 
-//  (       )
-void ControllerGUI::textEditorReturnKeyPressed (TextEditor& editor)
+void ControllerGUI::textEditorReturnKeyPressed (juce::TextEditor& editor)
 {
-    // Bouml preserved body begin 0004180D
-    // Bouml preserved body end 0004180D
 }
 
-void ControllerGUI::textEditorEscapeKeyPressed (TextEditor& editor)
+void ControllerGUI::textEditorEscapeKeyPressed (juce::TextEditor& editor)
 {
-    // Bouml preserved body begin 0004188D
-    // Bouml preserved body end 0004188D
 }
 
-void ControllerGUI::textEditorFocusLost (TextEditor& editor)
+void ControllerGUI::textEditorFocusLost (juce::TextEditor& editor)
 {
-    // Bouml preserved body begin 0004190D
-    // Bouml preserved body end 0004190D
 }
 
-void ControllerGUI::labelTextChanged (Label* labelThatHasChanged)
+void ControllerGUI::labelTextChanged (juce::Label* labelThatHasChanged)
 {
-    // Bouml preserved body begin 00048A0D
     if (labelThatHasChanged == value)
     {
-        controller->setValue (roundToInt (value->getValue()));
+        controller->setValueInSelectedScenes (juce::roundToInt (value->getValue()));
```

Comment:
Because the original function "setValue" was prefixed with the comment "//set value for selected Scenes" and I prefer a longer name that at least tells  little bit more what it actually does than a comment which is more prone to get outdated.

Reply:
seems sensible 👍🏻